README.md

```diff
@@ -46,7 +47,9 @@ Ready-made Enhydris development environment
 8. Once in the container, `dbimport` will import the database.
 
 9. `python manage.py migrate` will ensure the database is up-to-date (the
-   database import file could be slightly out of date).
+   database import file could be slightly out of date). If some of the migrations
+   failed due to existing tables that are already created by the `dbimport` command,
+   `python manage.py migrate --fake-initial` will mark the migrations as applied.
 
```

Comment:
This is a very rare case. I suspect that in this case it happened because in the recent big timezone changes enhydris-synoptic had some database changes that required manual intervention. If I had remembered this I would have created a new dump. So I suggest to remove this change.

Reply:
That makes sense. I will remove it.